static/js/src/cve/cve.js

```diff
@@ -71,3 +71,54 @@ searchInput.addEventListener("keyup", handleSearchInput);
 attachEvents();
 handleButtons();
 disableSelectedVersions();
+
+const priorities = {
+  "critical": "<b>Critical:</b> A world-burning problem that is exploitable for most Ubuntu users. Examples include remote root privilege escalations or remote data theft.",
+  "high": "<b>High:</b> Exploitable for many users in the default configuration of the affected software. Examples include serious remote denial of service of the system, local root privilege escalations or local data theft.",
+  "medium": "<b>Medium:</b> Exploitable for many users of the affected software. Examples include network daemon denial of service, cross-site scripting and gaining user privileges.",
+  "low": "<b>Low:</b> Does very little damage or is otherwise hard to exploit, due to small user base or other factors such as requiring specific environment, uncommon configuration, or user assistance. These tend to be included in security updates only when higher priority issues require an update or if many low-priority issues have built up.",
+  "negligible": "<b>Negligible:</b> May be a problem, but does not impose a security risk due to various factors. Examples include when the vulnerability is only theoretical in nature, requires a very special situation, has almost no install base or does no real damage. These typically will not receive security updates unless there is an easy fix and some other issue causes an update.",
+  "unknown": "<b>Unknown:</b> Open vulnerability where the priority is currently unknown and needs to be triaged.",
+}
+
+const statuses = {
+  "not-affected": "<b>Not vulnerable:</b> Packages which do not exist in the archive, are not affected by the vulnerability or have a fix applied in the archive.",
+  "pending": "<b>Pending:</b> A fix has been applied and updated packages are awaiting arrival into the archive. For example, this might be used when wider testing is requested for the updated package.",
+}
+
+const tooltipIconList = document.querySelectorAll('.cve-tooltip-icon')
+
+tooltipIconList.forEach(function(tooltipIcon) {
+  tooltipIcon.addEventListener("mouseover", function( event ) {
+    if (tooltipIcon.parentElement.querySelector(".cve-tooltip") == null) {
+      const priority = this.getAttribute('data-priority')
```

Comment:
instead of `this.getAttribute()` you can do `this.dataset.priority`

Reply:
Done.